src/stories/dagshub/data-engine/singlefileViewModal/SingleFileViewModal.stories.tsx

```diff
@@ -0,0 +1,304 @@
+import {Meta, StoryFn} from '@storybook/react';
+import React from 'react';
+import SingleFileViewModal, {
+    singleFileViewModalProps
+} from "../../../../components/dagshub/data-engine/singleFileViewModal/SingleFileViewModal";
+
+const meta: Meta<singleFileViewModalProps> = {
+    title: 'DagsHub/Data-Engine/Modals/Single file view modal',
+    component: SingleFileViewModal
+};
+
+export default meta;
+
+const Template: StoryFn<typeof SingleFileViewModal> = (args) => (
+    <SingleFileViewModal {...args} />
+);
+
+export const singlefileViewModalImage: StoryFn<typeof SingleFileViewModal> =
+    Template.bind({});
+
+singlefileViewModalImage.args = {
+    closeModal: () => {
+        console.log("close modal")
+    },
+    setActiveSingleItem: () => {
+        console.log("set active single item")
+    },
+    activeSingleItemIndex: 0,
+    setActiveSingleItemIndex: () => {
+        console.log("set active single item index")
+    },
+    hasMoreItemsToFetch: false,
+    loadMoreItems: () => {
+        console.log("load more items")
+    },
```

Comment:
After you're done with the component changes, provide a `getNext` and `getPrevious` function with a mock list of 3 items for the example.